lemminx-maven/src/test/java/org/eclipse/lemminx/maven/test/ClientServerConnection.java

```diff
@@ -72,7 +74,12 @@ public ClientServerConnection() throws IOException {
 	}
 
 	public void stop() throws InterruptedException, ExecutionException {
-		languageServer.shutdown().get();
+		try {
+			languageServer.shutdown().get(1, TimeUnit.MINUTES);
```

Comment:
This was added just as a test and probably should be removed

Reply:
Adding this forced shutdown seems to stop the CI from going on forever. It looks like the language server isin't able to shutdown correctly. (This is most likely due to the shutdown of the extension not finishing properly, not the language server's fault)